bugbug/repository.py

```diff
@@ -216,15 +219,14 @@ def _transform(commit):
             type_ = ext
         obj["types"].add(type_)
 
-        try:
-            after = HG.cat([path.encode("utf-8")], rev=commit.node.encode("ascii"))
-        except hglib.error.CommandError as e:
-            if b"no such file in rev" in e.err:
-                after = b""
-            else:
-                raise
-
-        sizes.append(after.count(b"\n"))
+        if not stats["deleted"] and not stats["binary"]:
+            try:
+                after = HG.cat([path.encode("utf-8")], rev=commit.node)
+                sizes.append(after.count(b"\n"))
+            except hglib.error.CommandError as e:
+                print((e, stats))
```

Comment:
What error can we have here?

Reply:
If a file has been deleted but rs_pp didn't see it then we'll catch it here

Reply:
OKOK, can we remove the print statement?